Dianey-Villains Database.sql

```diff
@@ -10,3 +10,31 @@ updatedAt datetime default current_timestamp on update current_timestamp,
 deletedAt datetime, 
 Primary Key(name)
 );
+show tables;
+DROP USER IF EXISTS 'newUser'@'localhost';
```

Comment:
user was wrongdoers.  Needed to change throughout repo for the database calls to work